evaluate/task/testing/task.go

```diff
@@ -11,15 +11,14 @@ import (
 	"github.com/symflower/eval-dev-quality/evaluate/metrics"
 	metricstesting "github.com/symflower/eval-dev-quality/evaluate/metrics/testing"
 	"github.com/symflower/eval-dev-quality/language"
-	"github.com/symflower/eval-dev-quality/model"
 	evaltask "github.com/symflower/eval-dev-quality/task"
 	"github.com/zimmski/osutil"
 )
 
-type TestCaseTask struct {
+type TestCaseTask[ModelCapability any] struct {
```

Comment:
https://go.dev/doc/tutorial/generics#declare_type_constraint ?

Reply:
Does not work with interfaces having methods